tests/standalone/six.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+import six
+import sys
+
+MAXSIZE = sys.maxsize
+
+def dispatch_types(value):
```

Comment:
Well, not being named `test_*` still doesn't make the code being called, you realize that?